packages/create-cloudflare/src/cli.ts

```diff
@@ -168,6 +167,12 @@ const configure = async (ctx: C3Context) => {
 	addWranglerToGitIgnore(ctx);
 
 	await updatePackageScripts(ctx);
+	if (ctx.template.installWorkersTypes) {
```

Comment:
would `installWorkersTypes === true` and `skipWranglerTypegen === false` be valid? (i.e. both install and generate) ?

if not we could have have `workersTypes === "generated" | "installed# | undefined`